web/annotation_layer_builder.js

```diff
@@ -94,10 +94,11 @@ class AnnotationLayerBuilder {
    * @param {PageViewport} viewport
    * @param {Object} options
    * @param {string} intent (default value is 'display')
+   * @param {Array<Object>} inferredLinkAnnotations
```

Comment:
This must be marked as optional.
```suggestion
   * @param {Array<Object>} [inferredLinkAnnotations]
```